presto-phoenix/src/test/java/io/prestosql/plugin/phoenix/TestPhoenixIntegrationSmokeTest.java

```diff
@@ -178,6 +178,16 @@ public void testCaseInsensitiveNameMatching()
         assertQuery("SELECT Val1 FROM testcaseinsensitive where Val1 < 1.2", "SELECT 1.1");
     }
 
+    @Test
+    public void testMissingColumnsOnInsert()
+            throws Exception
+    {
+        executeInPhoenix("create table tpch.test_col_insert(pk varchar not null primary key, col1 varchar, col2 varchar)");
```

Comment:
Can we move all the SQL Keywords to Upper case

Reply:
Done.